cg/server/endpoints/orders.py

```diff
@@ -178,6 +178,8 @@ def submit_order(order_type: OrderType):
                 content=request_json, file_format=FileFormat.JSON
             ),
         )
+        request_json["project_type"] = order_type
```

Comment:
so this line will be removed right?

Reply:
yes!

Reply:
Or a bit unsure why the FE would need to send it actually? This should always be correct